TophatExtensions/TophatBitriseExtension/TophatBitriseExtension.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TophatBitriseExtension.swift
+//  TophatBitriseExtension
+//
+//  Created by Lukas Romsicki on 2024-11-28.
+//  Copyright © 2024 Shopify. All rights reserved.
+//
+
+import Foundation
+import TophatKit
+import SwiftUI
+
+@main
+struct TophatBitriseExtension: TophatExtension, ArtifactProviding, SettingsProviding {
+	static let title: LocalizedStringResource = "Bitrise"
+
+	static var artifactProviders: some ArtifactProviders {
+		BasicArtifactProvider()
+		BranchArtifactProvider()
+	}
```

Comment:
As more extensions come in, it could be nice to add the extension name or product name as a header to the provider picker to better separate them. I'm not actually sure if `Picker` supports that out of the box though. Could help if there are ever duplicates too.
<img width="156" alt="image" src="https://github.com/user-attachments/assets/ecda7a3f-5dd4-4b4a-93ab-0837bda8de57">